src/elements/button/src/index.tsx

```diff
@@ -55,7 +56,7 @@ export const Button: React.FC<Props> = ({
   ...props
 }) => {
   const { theme }: any = useThemeUI()
-  const variantStyle = theme.buttons?.variants[variant]
+  const variantStyle = get(theme, `buttons.variants${variant}`)
```

Comment:
I haven't tested this, but isn't this missing a dot?

Reply:
Yes, you're right, thanks! Fixing